index.js

```diff
@@ -107,6 +112,30 @@ app.delete('/ideas', (req, res) => {
     res.status(200).send('Ideas successfully deleted.');
 });
 
+//DONATION METHODS
+
+app.get('/donations', (req, res) => {
+    res.status(200).json(ideasDonationSum(req.body.ideaId));
+});
+
+app.get('/donations/sumDifference', (req, res) => {
+    res.status(200).json(ideasSumDifference(req.body.ideaId));
+});
+
+app.post('/donations', (req, res) => {
+    try {
+        createDonationsWithDataValidation(
```

Comment:
Can this function be named `createDonation`?

Reply:
As long as I have hands it can! Updated. Commited.